### PR TITLE
Add a bio for GitHub Teacher

### PR DESCRIPTION
**Pull request by githubteacher**
This Pull Request adds a bio for @githubteacher.  I think this would be a great addition to the project.

It is a great way of illustrating the GitHub Flow. :+1:

### CODE REVIEW

File: annieisms.md [hunk 1]
```diff
@@ -0,0 +1,6 @@
+# Annie
+
+**Name:** Annie
+
+**Occupation:** Learner
+
```
Comment (GregSawin) [inline annieisms.md]: Great

File: githubteacher.md [hunk 2]
```diff
@@ -0,0 +1,5 @@
+# GitHub Teacher
+
+**Name:** GitHub Teacher
```
Comment (githubteacher) [inline githubteacher.md:3]: This seems a bit redundant. :smiley:

File: clivehy.md [hunk 3]
```diff
@@ -0,0 +1,5 @@
+# clivehy
+
+**Name:** clivehy
```
Comment (gmacario) [inline clivehy.md:3]: :+1:

File: imjeffc.md [hunk 4]
```diff
@@ -0,0 +1,5 @@
+#imjeffc
+
+**Name:** Jeff
+
+**Occupation:** Student
```
Comment (christabelle) [inline imjeffc.md:5]: This is great!

File: annieisms.md [hunk 5]
```diff
@@ -0,0 +1,6 @@
+# Annie
+
+**Name:** Annie
```
Comment (gmacario) [inline annieisms.md:3]: Good!

File: githubteacher.md (hunk 2 above)
Comment (mrumsky) [inline githubteacher.md:3]: :+1:

**Comment by githubteacher** [comment]
:+1: 
:ship: it!

:shipit:

**Comment by Ebaid** [comment]
:+1:

**Comment by samueltxkhoo** [comment]
Test.

**Pull request merged**